website/package.json

```diff
@@ -15,6 +15,7 @@
     "test": "tests"
   },
   "scripts": {
+    "prebuild": "node lib/changelog/generate-changelog-markdown-files.mjs",
```

Comment:
I'm not super familiar with `pre` commands, can you elaborate a bit on how this will get run? Is it something we need to do ourselves or will it happen automatically somehow?

Reply:
when running `yarn build` (or `npm build`) it will actually run the `prebuild` instructions and then the `build` instructions

Reply:
I assume Vercel runs `yarn build` in the case of our `website`

Reply:
yes, what @alex-ju said

Reply:
from a purely technical standpoint no, they would be generated on the fly when the website is built
in this PR I have added them to have something for preview, and so that in local development the files are there and the website renders without complaining of missing files
the alternative, if we want to have always the latest version checked in the repo, is to hook into the action that runs the `version/release` scripts and checks in the results.

Reply:
👍  ok, that sounds good to me. thanks for the explanation.